uisdk/src/main/java/com/karhoo/uisdk/screen/booking/map/BookingMapView.kt

```diff
@@ -388,15 +388,15 @@ class BookingMapView @JvmOverloads constructor(context: Context,
     fun setNoBottomPadding() {
         googleMap?.setPadding(0, 0, 0, 0)
         recentreMapIfDestinationIsNull()
+        showLocateUserButton()
 
         animateLocateMeButton(R.dimen.spacing_small, R.integer.animation_duration_slide_out_or_in_quotes)
```

Comment:
Do we still need this animateLocateMeButton?

Reply:
Yes. This makes sure the button is in the correct position each time it is shown when the quote view disappears.